tests/functional/aws-node-sdk/test/bucket/getBucketPolicy.js

```diff
@@ -0,0 +1,93 @@
+const assert = require('assert');
+const { errors } = require('arsenal');
+const { S3 } = require('aws-sdk');
+
+const getConfig = require('../support/config');
+const BucketUtility = require('../../lib/utility/bucket-util');
+
+const bucket = 'getbucketpolicy-testbucket';
+const bucketPolicy = {
+    Version: '2012-10-17',
+    Statement: [{
+        Sid: 'test-id',
+        Effect: 'Allow',
+        Principle: '*',
+        Action: 's3:putBucketPolicy',
+        Resource: 'arn:aws:s3::getbucketpolicy-testbucket',
+    }],
+};
+const expectedPolicy = {
+    Sid: 'test-id',
+    Effect: 'Allow',
+    Principle: '*',
+    Action: 's3:putBucketPolicy',
+    Resource: 'arn:aws:s3::getbucketpolicy-testbucket',
+};
+
+// Check for the expected error response code and status code.
+function assertError(err, expectedErr, cb) {
+    if (expectedErr === null) {
+        assert.strictEqual(err, null, `expected no error but got '${err}'`);
+    } else {
+        assert.strictEqual(err.code, expectedErr, 'incorrect error response ' +
+            `code: should be '${expectedErr}' but got '${err.code}'`);
+        assert.strictEqual(err.statusCode, errors[expectedErr].code,
+            'incorrect error status code: should be 400 but got ' +
+            `'${err.statusCode}'`);
+    }
+    cb();
+}
+
+const describeSkipUntilImpl =
+    process.env.BUCKET_POLICY ? describe : describe.skip;
+
+describeSkipUntilImpl('aws-sdk test get bucket policy', () => {
+    let s3;
+    let otherAccountS3;
+
+    before(done => {
```

Comment:
Minor: we can remove the callback here since there are not async methods. Alternatively, it looks like we don't need to use the `before` here as these values are just defined once.